### DISCUSSION TITLE
Using PBE_64 POTCARS

### DISCUSSION THREAD

**Discussion by Luccerboi** [JaGeo/LobsterPy#355] [Q&A] [answered]
Hi, 

For my project I was wondering about the possibility of using LOBSTERpy with the new VASP PAW POTCARS.
The tutorial mentions that input sets generated by pymatgen only support the PBE_54 POTCARS. For my project I am also simulating Lanthanide containing structures for which the updated PBE_64 POTCARS are a huge improvement, so I was wondering if there is a way I can still use the newer POTCARS and do the automated LOBSTER analysis.

If I naively use LOBSTERpy in a VASP calculation using the PBE_64 POTCAR, will this lead to issues?
If so, is there some other workaround possible?

Thanks!

**Comment by JaGeo** [comment, marked as answer]
I think there two points to consider:

1. If the valence electrons in both POTCAR files are the same, our basis set choice should be okay. However, you should confirm this to ensure the correct choice of the basis functions.

2. The Lobster basis sets have been fitted to atomic orbitals from the 54 POTCAR files. This can affect the quality of the projections as well. As long as you do not see very poor projections, it should be acceptable. Good projections typically have charge spilling values around 2% or less, especially for semiconductors. This point is likely minor as long as the bases functions correspond to the valence orbitals.

I hope this helps!

**Reply by Luccerboi** [reply]
Thanks a lot for the quick response!
I'll check with these considerations. If I do find discrepancies between the _54 and _64 (Lanthanide) POTCARS, I'll report them here for other to find as well!

**Reply by JaGeo** [reply]
Thank you! It is definitely on our todo to update lobsterpy for all the latest vasp and lobster changes but things take time!😅
